src/VisualizationTests/HelixWatch3DViewModelTests.cs

```diff
@@ -721,27 +721,59 @@ public void Python_CreatesVisualizations()
         }
 
         [Test]
-        public void Display_ByGeometryColor_HasColoredMesh()
+        public void Display_FrozenNode_HasTransparentMesh()
         {
             OpenVisualizationTest("Display.ByGeometryColor.dyn");
+            RunCurrentModel();
+            Assert.AreEqual(BackgroundPreviewGeometry.Count(), 4);
```

Comment:
@QilongTang sorry for the late review but you should swap the arguments to `Assert.AreEqual` in all these cases as the first arg is always the expected value and the second is actual. If a test fails NUnit can report confusing failure messages due to this mistake.